Gems/Atom/RPI/Code/Source/RPI.Public/Pass/Pass.cpp

```diff
@@ -670,6 +674,12 @@ namespace AZ
 
             // Setup attachment sources...
 
+            auto checkAttachmentSourceMissing = [this](const char* attachmentSourceType, const PassAttachmentRef& passAttachmentRef)
+            {
+                AZ_Warning("Pass", passAttachmentRef.m_pass.IsEmpty() && passAttachmentRef.m_attachment.IsEmpty(), "Pass [%s] could not find %s [Pass '%s', Attachment '%s']",
+                    m_path.GetCStr(), attachmentSourceType, passAttachmentRef.m_pass.GetCStr(), passAttachmentRef.m_attachment.GetCStr());
```

Comment:
```suggestion
                AZ_Warning("Pass", !passAttachmentRef.m_pass.IsEmpty() && !passAttachmentRef.m_attachment.IsEmpty(), "Pass [%s] could not find %s [Pass '%s', Attachment '%s']",
                    m_path.GetCStr(), attachmentSourceType, passAttachmentRef.m_pass.GetCStr(), passAttachmentRef.m_attachment.GetCStr());
```

Reply:
This check will only be performed when an attachment source was not found. If an attachment source was not found because the user didn't attempt to specify one, it isn't an error. It's only an error if the user specified a pass and/or attachment name.

I'll try to update the code to make this more clear.